### DISCUSSION TITLE
Easiest way to get the output from sep.extract in J2000 coordinates

### DISCUSSION THREAD

**Discussion by HannoSpreeuw** [sep-developers/sep#145] [Q&A]
What would be the easiest way to get the output of sep.extract in an astronomical coordinate system, like J2000? It seems all output from sep.extract is in pixel coordinates.

I guess the positions of sources from sep.extract could be converted from pixel coordinates to J2000 using an Astropy routine. Might not be superfast, but okay.

More difficult to convert would be errors and ellipse parameters. 
Source Extractor does, however, provide for this.

**Comment by alexw-im** [comment]
It sounds like what you want is a plate solve, image in point coordinates out. That problem is solved in a couple steps:
1. Extract sources in pixel coordinates
2. Calculate which stars those sources are likely to be
3. Calculate a best-fit solution of our image pose (and thus all the individual sources in the image) vs the many input samples (each catalog stare has its own known coordinates, some of which will agree more or less with the overall image estimate) 

This library is for doing step 1 alone. To do steps 2 and 3, you want something along the lines of Astrometry.net, for which there exists a convenient looking python library to perform any specific tasks you might want: https://github.com/neuromorphicsystems/astrometry
